src/main/java/teammates/common/util/Const.java

```diff
@@ -1403,6 +1403,9 @@ private Const() {
         public static final String STUDENT_COURSE_JOIN_SUCCESSFUL = "You have been successfully added to the course %s.";
         
         /** Template String. Parameters:  Course ID */
+        public static final String DELETED_STUDENT_ATTEMPTING_TO_JOIN =
```

Comment:
Rename this constant as well, to `NON_EXISTENT_STUDENT_ATTEMPTING_TO_JOIN_COURSE`